src/app/ReadClient.cpp

```diff
@@ -873,8 +1004,8 @@ void ReadClient::OnResubscribeTimerCallback(System::Layer * apSystemLayer, void
 {
     ReadClient * const _this = reinterpret_cast<ReadClient *>(apAppState);
     assert(_this != nullptr);
-    _this->SendSubscribeRequest(_this->mReadPrepareParams);
     _this->mNumRetries++;
+    _this->SendSubscribeRequest(_this->mReadPrepareParams);
```

Comment:
Why this change?

Reply:
this is accidental change, removed, thanks